java-manta-client-unshaded/src/main/java/com/joyent/manta/client/MantaClient.java

```diff
@@ -358,15 +358,27 @@ public void delete(final String rawPath) throws IOException {
      * @throws IOException If an IO exception has occurred.
      * @throws MantaClientHttpResponseException If a HTTP status code other than {@code 200 | 202 | 204} is encountered
      */
-    public void delete(final String rawPath,
-                       final MantaHttpHeaders requestHeaders) throws IOException {
-        Validate.notBlank(rawPath, "rawPath must not be blank");
+    public void delete(final String rawPath, final MantaHttpHeaders requestHeaders) throws IOException {
+        delete(rawPath, requestHeaders, config.getPruneEmptyParentDepth());
+    }
 
+    /**
+     * Deletes several directories in manta.
+     * @param paths - A list of paths to be deleted.
+     * @param requestHeaders - headers can be null.
+     * @throws IOException If a HTTP status code other than {@code 200 | 202 | 204} is encountered
```

Comment:
Missing `@param pruneDepth`.

It seems checkstyle is ignoring JavaDocs for package-private methods, I'll see how we can fix that.